frontend/src/metabase/admin/permissions/utils/graph/permissions-diff.ts

```diff
@@ -1,7 +1,7 @@
 import _ from "underscore";
 
 import { Group, GroupsPermissions } from "metabase-types/api";
-import Database from "metabase-lib/lib/metadata/Database";
+import { Database } from "metabase-types/types/Database";
```

Comment:
Perhaps @alxnddr can help me with this one. 

The database objects in this file are not actually instances of the `metabase-lib` `Database`, so I've needed to change which type we are using.

The type that matches this code best seems to be the deprecated `Database` in `metabase-types/types/Database`, but I also had to add `schema_name` to the `Table` type in `metabase-types/types/Table` to make types in this file pass the check.

Does this seem right?

Reply:
Iit uses custom subset of `Database` so probably we should define a custom type here `frontend/src/metabase/admin/permissions/selectors/data-permissions/diff.ts`
As far as I rememebr, hidden tables did not get there that's why I created a new selector